src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,101 @@
+package controller;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import domain.Lotto;
+import domain.Place;
+import service.LottoService;
+import view.InputView;
+import view.OutputView;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class LottoController {
+    private LottoService lottoService;
+    public static final int LOTTO_PRICE = 1000;
+    public static final Long FIFTH = 5000L;
+    public static final Long FORTH = 50000L;
+    public static final Long THIRD = 1500000L;
+    public static final Long SECOND = 30000000L;
+    public static final Long FIRST = 2000000000L;
```

Comment:
이런 식으로 숫자의 자리수가 많아질 때는 `_`를 사용해 자리수를 구분시켜 줄 수 있습니다!
```suggestion
    public static final Long FIRST = 2_000_000_000L;
```